llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -1923,8 +1923,9 @@ isLoopVariantIndirectAddress(ArrayRef<const Value *> UnderlyingObjects,
 // of various temporary variables, like A/BPtr, StrideA/BPtr and others.
 // Returns either the dependence result, if it could already be determined, or a
 // tuple with (Distance, Stride, TypeSize, AIsWrite, BIsWrite).
-static std::variant<MemoryDepChecker::Dependence::DepType,
-                    std::tuple<const SCEV *, uint64_t, uint64_t, bool, bool>>
+static std::variant<
+    MemoryDepChecker::Dependence::DepType,
+    std::tuple<const SCEV *, uint64_t, uint64_t, uint64_t, bool, bool>>
```

Comment:
Comment above needs updating -- though I feel like at this point this tuple really wants to be a struct.

Reply:
Done in cac4c14ecfcb892201ae8f4b1559909cc22082d0 and updated this PR, thanks!